Sources/SwiftQueue/SwiftQueueManager.swift

```diff
@@ -105,6 +105,26 @@ public final class SwiftQueueManager {
         }
     }
 
+    /// number of jobs with this tag in all queues
+    public func jobCount(tag: String) -> Int {
+        assertNotEmptyString(tag)
+        var count = 0
+        for element in manage.values {
+            count += element.operationCount(tag: tag)
+        }
```

Comment:
This part could also be written as 

```swift
for case let item as SqOperation in element.operations where item.info.tags.contains(tag) {
    count += 1
}
```

Then we don't need to change the code of SqOperation